package.json

```diff
@@ -87,9 +50,6 @@
     "typescript": "^4.9.5",
     "yargs": "^17.3.1"
   },
-  "files": [
-    "dist/**/*"
-  ],
   "resolutions": {
```

Comment:
should we move resolutions for runtime dependencies to the package's package.json?